apps/ftu/js/navigation.js

```diff
@@ -246,6 +249,16 @@ var Navigation = {
             sharePerformance.checked = req.result[settingName] || false;
           };
         }
+
+        // If it's a dogfooder, we don't want them to disable the metrics.
+        var dogfood = settings && settings.createLock().get(DOGFOODSETTING);
+        if (dogfood) {
+          dogfood.onsuccess = function() {
+            if(dogfood.result[DOGFOODSETTING] === '1') {
```

Comment:
You are right.  I was thinking it was weird too.  I will change it in my other patches and just do true/false here.